spec.md

```diff
@@ -1019,11 +1037,12 @@ The implementation MAY impose a maximum limit and return a partial set with pagi
 
 The following parameters SHOULD be specified on the request:
 
-| Name            | Kind   | Description                                                    |
-|-----------------|--------|----------------------------------------------------------------|
-| `Host`          | header | Standard HTTP Host Header. SHOULD be set to the registry host. |
-| `Authorization` | header | An RFC7235 compliant authorization header.                     |
-| `name`          | path   | Name of the target repository.                                 |
+| Name            | Kind   | Description                                                       |
+|-----------------|--------|-------------------------------------------------------------------|
+| `Host`          | header | Standard HTTP Host Header. SHOULD be set to the registry host.    |
+| `Authorization` | header | An RFC7235 compliant authorization header.                        |
+| `name`          | path   | Name of the target repository.                                    |
+| `ns`            | query  | (OPTIONAL) Namespace of repository. SHOULD be set to source host. |
```

Comment:
should we clarify that if `ns` is _not_ provided, there is a default assumption for the source host? or is that implied in the configuration of registry-proxy?